moveit_core/robot_model/CMakeLists.txt

```diff
@@ -12,20 +22,24 @@ add_library(${MOVEIT_LIB_NAME}
   src/revolute_joint_model.cpp
   src/robot_model.cpp
   )
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+# set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
 
-target_link_libraries(${MOVEIT_LIB_NAME} moveit_profiler moveit_exceptions moveit_kinematics_base ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(${MOVEIT_LIB_NAME} moveit_profiler moveit_exceptions moveit_kinematics_base ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
+# add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  urdf
+  urdfdom_headers
+  visualization_msgs)
 
-if(CATKIN_ENABLE_TESTING)
+if(AMENT_ENABLE_TESTING)
```

Comment:
Did anyone ever propose to just call it `ENABLE_TESTING` in both frameworks?

Reply:
@v4hn this debate is outside the scope of this PR. In the interest of porting MoveIt quickly, please start those discussions in the ros2 repos or Discourse.

Reply:
You should use CMake's standard `BUILD_TESTING`: https://github.com/ros2/rclcpp/blob/ef5f3d3fc15096d4bf41d0e7e42849b18e9408a4/rclcpp/CMakeLists.txt#L137

Reply:
Thanks @wjwwood, 700c45c2fcb6bbab76ce2eb745cf49dec4e3b04b addresses this but has been commented for now until we have all dependencies ported